storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,293 @@
+
+| Endpoint                                    | Where to find this endpoint in the console       |Notes                                                        |
+|----------------------------------------------|-------------------------------------------------|-------------------------------------------------------------|
+| `https://whatever-i-want.beautiful-domain.com`| The **Edge Services** tab  (after customization) | Edge Services serves bucket content when this endpoint is used. The pre-existing endpoints shown in the table in the previous section also continue to be functional as before. |
+
+The procedure for adding a customized endpoint is as follows:
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Configure domain**. The following screen displays:
+    
+    TODO SCREENSHOT
+
+5. Set a subdomain from which your bucket will be accessible. You must already own the primary domain. For example,if you own `beautiful-domain.com`, choose any subdomain you like and and enter `my-chosen-subdomain.beautiful-domain.com` into the box.
+
+6. Log into your domain provider, and locate the DNS settings for your domain. Create a new CNAME record pointing your subdomain to the Edge Services endpoint for your bucket displayed in the Scaleway console. For help setting up CNAME records and troubleshooting any problems, [check out our dedicated documentation](/storage/object/reference-content/cname-record/)
+
+7. Back in the Scaleway console, click the `Verify CNAME` button to check whether your CNAME record has been correctly configured. Edge Services will carry out a check, and if it is successful the following message displays:
+
+      <Lightbox src="scaleway-cname-success.webp" alt="Verification passed: Your DNS record has been correctly configured" />
+
+      If the check fails, an error message will display. See the documentation linked above for help troubleshooting such errors.
+
+8. Provide an SSL/TS certificate for your subdomain so that it can serve traffic over HTTPS. You have two options for this:
+    - Select an existing certificate that you have stored in [Scaleway Secret Manager](/identity-and-access-management/secret-manager/quickstart/).
+    - Manually import a certificate into Scaleway Secret Manager:
+      - Enter a name for your certificate (alphanumeric characters only)
+      - Optionally, add tags by typing each tag and then pressing enter
+      - Copy and paste the full PEM-formatted certificate chain into the box.
+      Your certificate will be automatically stored in Secret Manager and billed accordingly. For Edge Services Private Beta testers, a credit of TODO will be applied to your Scaleway account for Secret Manager storage.
```

Comment:
@RoRoJ we eventually decided that we won't give a voucher for this since the cost is too low. We will let the users know that Secret Manager usage will be billed in the enrollment emails. Maybe we can add an estimate of the costs incurred here. Based on Secret Manager pricing the cost will be somewhere around 0,07€/month/bucket exposed (if they need to add versions which is quite unlikely) they will go up to 0,25€/month/bucket